src/lib.rs

```diff
@@ -169,6 +169,7 @@ mod context;
 pub mod constants;
 pub mod ecdh;
 pub mod key;
+pub mod schnorrsig;
```

Comment:
I'd like the module to be called `bip340` rather than `Schnorr`. Both as a dig at Schnorr for patenting things, and also to be clear that this is a specific Schnorr-based signature scheme specified in BIP340.

Reply:
Schnorr has been removed from this module :).  Only mentions left are in the sys module as the method calls of secp use it.